test/integration/apiserver/patch_test.go

```diff
@@ -128,6 +131,66 @@ func TestPatchConflicts(t *testing.T) {
 
 }
 
+func TestPatchDeletionRetainKey(t *testing.T) {
+	s, clientSet, closeFn := setup(t)
+	defer closeFn()
+
+	ns := framework.CreateTestingNamespace("deletion-patch-retain-key", s, t)
+	defer framework.DeleteTestingNamespace(ns, s, t)
+
+	deploymentName := "noxu-deployment"
+	podName := "noxu-pod"
+	containerName := "i-almost-forget-noxu"
```

Comment:
what does `noxu` mean anyway? I assumed it was a made-up word by @deads2k in the integration tests for apiextensions-apiserver but I see it everywhere now. :)

Reply:
<img width="288" alt="screen shot 2018-08-21 at 11 58 16 am" src="https://user-images.githubusercontent.com/12049380/44379656-91a48f80-a539-11e8-9f09-c9cb59bcec0a.png">